android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java

```diff
@@ -71,6 +71,7 @@
     public static boolean isNotification;
 
     private static final String GOOGLE = "Google";
+    public static final String BRAVE = "Brave";
```

Comment:
@deeppandya is this the string that gets shown in the UI for Onboarding? We need to change to `Brave Search beta` but I wanted to confirm before changing

Reply:
Updated! 😄